docs/guides/install-qiskit.mdx

```diff
@@ -75,19 +75,31 @@ Whether you will work locally or in a cloud environment, the first step for all
          <OperatingSystemTabs>
            <TabItem value="mac" label="macOS">
              ```shell
-             source /path/to/virtual/environment/bin/activate
+             source .venv
              ```
            </TabItem>
 
            <TabItem value="linux" label="Linux">
              ```shell
-             source /path/to/virtual/environment/bin/activate
+             source .venv
              ```
            </TabItem>
 
            <TabItem value="win" label="Windows">
+              If using PowerShell:
+
+             ```text
+             .venv\Scripts\Activate.ps1
+             ```
+             If using Git Bash:
+
+             ```text
+             .venv/source/Activate
+             ```
+              If using command prompt:
+
              ```text
-             c:\path\to\virtual\environment\Scripts\Activate.ps1
+             .venv\Scripts\activate
```

Comment:
Did you try this? I'm not sure what it should be for command prompt

Reply:
yes